csharp/build.sh

```diff
@@ -37,7 +37,7 @@ export CONFIGURATION=$STEP
 
 export STEP=$CONFIGURATION
 
-xbuild /p:Configuration=$CONFIGURATION $XBUILDOPT $PROJ
+xbuild /p:Configuration=$CONFIGURATION;AllowUnsafeBlocks=true $XBUILDOPT $PROJ
```

Comment:
"xbuild /p:Configuration=$CONFIGURATION;AllowUnsafeBlocks=true" can be error on Linux as the semicolon ";" is treated as the separator of two commands.
That's why compilation failed : [./build.sh: line 40: /verbosity:minimal: No such file or directory]( https://travis-ci.org/Microsoft/Mobius/builds/139925856#L2208)